### ISSUE TITLE
Optimize header image design & placement

### ISSUE THREAD

**Issue by kpshek** [cerner/cerner.github.io#26]
- Header images need to be resized for responsive scaling. (waiting on size recommendations based on development capabilities.)
- These were intended to be animated and could be converted easily once a size is settled on. Not a need, but nice to have.
- New images should also consider title placement so that it can be centered horizontally and legible.
- Header images need to be shortened height-wise so that they don't take up so much of the screen (today, the use must scroll a lot to read the content)

**Comment by Kallie-Riff** [comment]
I got a link for the mockups! 

Here is the before: 
![image](https://user-images.githubusercontent.com/7071773/30074889-c0854c5a-9238-11e7-8d5d-ace3a2ef68b6.png)

Here is my proposed after: 
![image](https://user-images.githubusercontent.com/7071773/30074924-df423f68-9238-11e7-852d-50bffde4e101.png)

This change is going to be pretty easy! It's basically just swapping out the images. No pesky CSS changes with this one (My virtual machine doesn't update when I change CSS).


Thoughts, comments, suggestions?

**Comment by LindsayUllyot** [comment]
@Kallie-Riff This looks fantastic and improves the overall look of the page. Minimizing the size of the photos and making content more visible is what we were looking for. @kpshek, if there is anything else you need prior to publishing, please let us know. Thank you!
